config/services.yaml

```diff
@@ -164,6 +164,10 @@ services:
             string $downloadBaseDir: '%env(DOWNLOAD_BASE_DIRECTORY)%'
             string $downloadBaseUrl: '%env(DOWNLOAD_BASE_URL)%'
 
+    App\Controller\UI\DatalandController:
+        bind:
+            string $issueTrackingBaseUrl: '%env(ISSUE_TRACKING_BASE_URL)%'
```

Comment:
I don't this is the right link, wouldn't work. We need to use `https://griidc.atlassian.net/browse/` as the prefix for the URL

Reply:
yeah, I saw that in our .env we have it set to ISSUE_TRACKING_BASE_URL='https://griidc.atlassian.net/issues' and figured it just needed updating. Are they 2 different things?

Reply:
I updated the .env accordingly so the correct default is used. Evidently, at some point in the past Atlassian changed /issue to /browser, I guess.